dangerfile.js

```diff
@@ -133,3 +134,8 @@ markdown.forEach(path => {
       });
     });
 });
+
+commonFileWarnings('test.log', {
+  msg:
+    'The build logs contain these warnings (check the [Travis](https://travis-ci.org/Skyscanner/backpack) build output for more details):',
+});
```

Comment:
New Danger rule to show warnings from the build: https://github.com/sogame/danger-plugin-toolbox/blob/master/docs/validations.md#commonfilewarnings

This is an example output:
<img width="775" alt="warnings" src="https://user-images.githubusercontent.com/7152781/57108374-d3c1da00-6d32-11e9-919f-580bcb137d92.png">